ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -416,11 +418,13 @@ export default function BuildQuote({
                       <a
                         className="build-quote__token-etherscan-link build-quote__underline"
                         key="build-quote-etherscan-link"
-                        href={`https://etherscan.io/token/${selectedToToken.address}`}
+                        href={`${rpcPrefs.blockExplorerUrl}/token/${selectedToToken.address}`}
                         target="_blank"
                         rel="noopener noreferrer"
                       >
-                        {t('etherscan')}
+                        {rpcPrefs.blockExplorerUrl
+                          ? rpcPrefs.blockExplorerUrl
```

Comment:
Hmm - this seems OK for now, but we should come up with some other text to go here other than the URL.

Maybe for now we could at least limit it to the hostname, rather than the full URL? That might look a bit nicer.

```suggestion
                          ? new URL(rpcPrefs.blockExplorerUrl).hostname
```

Reply:
Done in 7a5b0dd92